lib/ansible/modules/utilities/logic/assert.py

```diff
@@ -38,6 +38,10 @@
     version_added: "2.7"
     description:
       - "The customized message used for a successful assertion"
+  quiet:
+    version_added: "2.8"
+    description:
+      - "Set this to true to avoid verbose output"
```

Comment:
```suggestion
      - "Set this to C(true) to avoid verbose output."
```
(Also see [here](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation).)

Reply:
Good idea.  I also added . to the end of the existing descriptions, as https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-block specifies: "description: Detailed explanation of what this option does. It should be written in full sentences."